transport/azure-servicebus/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+plugins {
+    // Apply precompiled plugins.
+    id("ort-server-kotlin-jvm-conventions")
+    id("ort-server-publication-conventions")
+}
+
+group = "org.eclipse.apoapsis.ortserver.transport"
+
+dependencies {
+    implementation(projects.transport.transportSpi)
+    implementation(projects.utils.logging)
+
+    implementation(libs.kotlinxCoroutines)
+    implementation(libs.kotlinxCoroutinesSlf4j)
+    implementation(libs.azureIdentity)
+    implementation(libs.azureMessagingServicebus)
+
+    runtimeOnly(libs.logback)
+
+    testImplementation(projects.model)
+    testImplementation(testFixtures(projects.config.configSpi))
+    testImplementation(testFixtures(projects.transport.transportSpi))
+
+    testImplementation(libs.kotestAssertionsCore)
+    testImplementation(libs.kotestExtensionsTestContainer)
+    testImplementation(libs.kotestRunnerJunit5)
+    testImplementation(libs.mockk)
+    testImplementation(libs.testContainers)
+    testImplementation(libs.testContainersRabbitMq)
```

Comment:
Is the Rabbit MQ stuff a left-over? Hint: You might want to run the `projectHealth` task on this new project.

Reply:
I have removed a few unused dependencies.